lib/src/model.dart

```diff
@@ -1018,7 +1018,7 @@ class Class extends ModelElement {
       name.hashCode, library.name.hashCode, library.package.name.hashCode);
 
   @override
-  String get _href => '${library.nameForFile}/$fileName';
+  String get _href => '${library.fileName}/$fileName';
```

Comment:
This is why `fileName` didn't make sense... because we also use it here for a path part.